### PR TITLE
FIX: Changed name of extension to be compatible with PHP7

### PR DESCRIPTION
**Pull request by mi3ll**

### CODE REVIEW

**Comment by jonom** [comment]
Thanks @mi3ll, I'm on vacation for another week but I'll test this when I get back

File: _config/config.yml [hunk 1]
```diff
@@ -7,4 +7,8 @@ LeftAndMain:
     - 'betternavigator/javascript/BN.LeftAndMain.Preview.js'
 ContentController:
   extensions:
-    - BetterNavigator
+    - BetterNavigatorExtension
+
+Injector:
```
Comment (jonom) [inline _config/config.yml]: I think let's remove this part, probably no one is using BetterNavigatorExtension on anything besides ContentController  and I need to bump the major version number due to another contribution anyway, so let's include this at the same time and not worry about backwards compatibility here.

Comment (mi3ll) [inline _config/config.yml]: Fixed in last commit @b202b2e

**Comment by 3Dgoo** [comment]
@jonom have you had a chance to look at this yet?

**Comment by jonom** [comment]
Sorry, forgot all about it. @mi3ll could you please squash your commits and I'll merge

**Comment by mi3ll** [comment]
Sorry, completely forgot about this. I have now squashed it.

**Comment by jonom** [comment]
Thanks @mi3ll

**Pull request merged**